lib/Logging/LogRepository.php

```diff
@@ -12,4 +12,27 @@ class LogRepository extends Repository {
   protected function getEntityClassName() {
     return LogEntity::class;
   }
+
+  /**
+   * @param \DateTimeInterface|null $dateFrom
+   * @param \DateTimeInterface|null $dateTo
+   * @return LogEntity[]
+   */
+  public function getLogs(\DateTimeInterface $dateFrom = null, \DateTimeInterface $dateTo = null): array {
+    $query = $this->doctrineRepository->createQueryBuilder('l')
+      ->select('l');
+
+    if ($dateFrom instanceof \DateTimeInterface) {
+      $query
+        ->where('l.createdAt > :dateFrom')
+        ->setParameter('dateFrom', $dateFrom->format('Y-m-d H:i:s'));
+    }
+    if ($dateTo instanceof \DateTimeInterface) {
+      $query
+        ->andWhere('l.createdAt < :dateTo')
+        ->setParameter('dateTo', $dateTo->format('Y-m-d H:i:s'));
+    }
+
+    return $query->getQuery()->getResult();
```

Comment:
We don't have any other limit than dates here. I think that there might be cases that there will be many logs (e.g. some blocked cron that generate a bunch of logs every 30 seconds). It was not in the specification but we could add limit and offset inputs + a number of rows displayed to the filter area so that we are able somehow "paginate". What do you think? cc @NeosinneR  My concern is that in some cases we won't be able to render the default recent 7 days.

Reply:
The specification says: 
> No pagination, show all available results

Reply:
I wanted to skip pagination to keep the ticket as simple as possible. What do you think would be the reason to fail to render the page? The table is very simple, so I think even a few megabytes of logs should be renderable (although it might take some time on slower connections).

Reply:
The table is rendered by React and I'm not sure about performance with e.g. 10K of rows. On the other hand, this is not for users so in some probably rare cases we can live with a slow page and we can always add some paginations later or use some "[virtual table](https://bvaughn.github.io/react-virtualized/#/components/Table)" later.

Reply:
I am not sure if react will have issues, but the whole DB query, processing the log, and passing it to the template may time out.

Reply:
Is it difficult to add those offset and limit inputs (with defaults 0, 100) and apply them? It is not super UX for pagination but could be enough for digging in logs and could protect us from potential issues.

Reply:
I guess not that much, I would just pass them with the other existing inputs.